contexts/credentials/v2

```diff
@@ -43,17 +44,20 @@
     },
     "cnf": {
       "@id": "https://www.iana.org/assignments/jwt#cnf",
-      "@context": {
-        "@protected": true,
-        "kid": {
-          "@id": "https://www.iana.org/assignments/jwt#kid",
-          "@type": "@id"
-        },
-        "jwk": {
-          "@id": "https://www.iana.org/assignments/jwt#jwk",
-          "@type": "@json"
+      "@context": [
+        null,
```

Comment:
Not easy to suggest this in github right now, but want it tracked, so don't apply directly:

```suggestion
```

Reply:
Fixed in c44da90e4d0be776e8f7dc465bfee1b87fd4d0fc